Openal build fails due to unknown module

```shell
┌ openal-soft git  ....................................... [Updates found]
├ Running pre cmake from openal-soft_extra.sh...
├ Running cmake...
├ Running build...
Likely error (tail of the failed operation logfile):
[165/214] Building CXX object CMakeFiles/OpenAL.dir/alc/alc.cpp.o
ninja: build stopped: subcommand failed.
[1/50] Building CXX object CMakeFiles/OpenAL.dir/alc/effects/fshifter.cpp.o
FAILED: [code=1] CMakeFiles/OpenAL.dir/alc/effects/fshifter.cpp.o
E:\MABS\local64\bin\g++.bat -DALC_API="" -DAL_ALEXT_PROTOTYPES -DAL_API="" -DAL_BUILD_LIBRARY -DAL_LIBTYPE_STATIC -DNOMINMAX -DWIN32_LEAN_AND_MEAN -D_GLIBCXX_ASSERTIONS=1 -D_WIN32 -D__USE_MINGW_ANSI_STDIO -IE:/MABS/build/openal-soft-git/gsl/include -IE:/MABS/build/openal-soft-git/include -IE:/MABS/build/openal-soft-git/build-64bit -IE:/MABS/build/openal-soft-git -IE:/MABS/build/openal-soft-git/common -IE:/MABS/build/openal-soft-git/fmt-11.2.0/include -D_FORTIFY_SOURCE=2 -fstack-protector-strong -mtune=generic -O2 -pipe -D__USE_MINGW_ANSI_STDIO=1 -O3 -DNDEBUG -std=gnu++20 -Wunused -Wall -Wextra -Wshadow -Wconversion -Wcast-align -Wpedantic -Wformat -Wformat=2 -Wold-style-cast -Wnon-virtual-dtor -Woverloaded-virtual -Wtrampolines -Wbidi-chars=any -Wno-interference-size -Werror=undef -Werror=format-security -ftrivial-auto-var-init=pattern -fno-math-errno -MD -MT CMakeFiles/OpenAL.dir/alc/effects/fshifter.cpp.o -MF CMakeFiles\OpenAL.dir\alc\effects\fshifter.cpp.o.d -fmodules-ts -fmodule-mapper=CMakeFiles\OpenAL.dir\alc\effects\fshifter.cpp.o.modmap -MD -fdeps-format=p1689r5 -x c++ -o CMakeFiles/OpenAL.dir/alc/effects/fshifter.cpp.o -c E:/MABS/build/openal-soft-git/alc/effects/fshifter.cpp
E:/MABS/build/openal-soft-git/alc/effects/fshifter.cpp:47:1: fatal error: unknown compiled module interface: no such module
   47 | import window.hann;
      | ^~~~~~
compilation terminated.
ninja: build stopped: subcommand failed.
build failed. Check E:/MABS/build/openal-soft-git/build-64bit/ab-suite.build.log
This is required for other packages, so this script will exit.
  Creating diagnostics file...

Attach E:\MABS\build\logs.zip to the GitHub issue.
Make sure the suite is up-to-date before reporting an issue. It might've been fixed already.

Try running the build again at a later time.

``` 

Cheers
John

[logs.zip](https://github.com/user-attachments/files/25828028/logs.zip)

--- a/build/media-suite_deps.sh
+++ b/build/media-suite_deps.sh
@@ -86,7 +86,7 @@ SOURCE_REPO_MINIZIPNG=https://github.com/zlib-ng/minizip-ng.git
 SOURCE_REPO_MPV=https://github.com/mpv-player/mpv.git
 SOURCE_REPO_MUJS=https://codeberg.org/ccxvii/mujs.git
 SOURCE_REPO_NEON=https://github.com/notroj/neon.git
-SOURCE_REPO_OPENAL=https://github.com/kcat/openal-soft.git
+SOURCE_REPO_OPENAL=https://github.com/kcat/openal-soft.git#tag=latest
 SOURCE_REPO_OPENAPV=https://github.com/AcademySoftwareFoundation/openapv.git
 SOURCE_REPO_OPENCLHEADERS=https://github.com/KhronosGroup/OpenCL-Headers.git
 SOURCE_REPO_OPUS=https://github.com/xiph/opus.git

